otterdog/eclipse-edc.jsonnet

```diff
@@ -191,6 +191,30 @@ orgs.newOrg('technology.edc', 'eclipse-edc') {
         },
       ],
     },
+    orgs.newRepo('Fleet') {
+      allow_rebase_merge: false,
+      allow_update_branch: false,
+      delete_branch_on_merge: false,
+      description: "Fleet Management Components",
+      has_discussions: true,
+      has_wiki: false,
+      gh_pages_build_type: "legacy",
+      gh_pages_source_branch: "gh-pages",
+      gh_pages_source_path: "/",
+      squash_merge_commit_title: "PR_TITLE",
+      web_commit_signoff_required: false,
+      workflows+: {
+        default_workflow_permissions: "write",
```

Comment:
According to #26 EF strongly advises to use read permissions by default with specific permissions as required. We agreed to do this for all new repositories some month ago. Thus, please update this property accordingly.

Reply:
Thx - fixed.